Objects/codeobject.c

```diff
@@ -234,9 +236,56 @@ PyCode_New(int argcount, int posonlyargcount, int kwonlyargcount,
     co->co_zombieframe = NULL;
     co->co_weakreflist = NULL;
     co->co_extra = NULL;
+
+    co->co_opcache_flag = 0;
+    co->co_opcache_size = 0;
+    co->co_opcache_map = NULL;
+    co->co_opcache = NULL;
```

Comment:
Purely as a style question ... why did you change the order of the fields? (as opposed to initializing in definition order)

Reply:
I didn't changed order in here... I changed only order of struct for compactness and
readability.  I didn't changed here because (a) I just missed it and (b) order in here is
not so important than order in struct.